.github/CODEOWNERS

```diff
@@ -9,6 +9,7 @@
 /python/sglang/srt/models @merrymercy @Ying1123 @hnyls2002 @zhyncs @ispobock @ByronHsu
 /python/sglang/srt/openai_api @merrymercy @Ying1123 @hnyls2002 @zhyncs @ispobock @ByronHsu
 /python/sglang/srt/sampling @merrymercy @hnyls2002
+/python/sglang/srt/speculative @Ying1123 @merrymercy @rkooo567 @kssteven418
```

Comment:
Can we give write permission to these two authors, or else we will encounter an error?

```
CODEOWNERS errors
Unknown owner on line 12: make sure @rkooo567 exists and has write access to the repository
```